pkg/artifactory/resource_artifactory_remote_cargo_repository.go

```diff
@@ -43,6 +50,7 @@ func unpackCargoRemoteRepo(s *schema.ResourceData) (interface{}, string, error)
 		RemoteRepositoryBaseParams: unpackBaseRemoteRepo(s, "cargo"),
 		RegistryUrl:                d.getString("git_registry_url", false),
 		AnonymousAccess:            d.getBool("anonymous_access", false),
+		ListRemoteFolderItems:      d.getBool("list_remote_folder_items", false),
```

Comment:
Add this to `unpackBaseRemoteRepo()` instead?

Reply:
Will move this attribute under RemoteRepositoryBaseParams.